hmcts/pre-api
# S28-2981/db remove recordings url

JIRA ticket(s)
[S28-2981](https://tools.hmcts.net/jira/browse/S28-2981)
Change description
Remove recordings.url from db
Ignore URL value in requests attempting to set the value
Return for URL value RecordingDTO is always null
Important

This PR requires manual testing by QA. Please notify the QA team @hmcts/pre-rec-evidence-qa.

QA instructions

Ensure db migration is non breaking (copy current staging + see the running migration on the db is non-breaking)
Make PUT /recordings/{id} request and set the value of URL in the request body- see no error (value is ignored) -> 20x response
Make GET /recordings request, see url is null
